packages/editor/package.json

```diff
@@ -55,6 +55,7 @@
     "@wepublish/karma.run-react": "^0.2.0",
     "apollo-upload-client": "^16.0.0",
     "array-move": "^2.2.1",
+    "client-compress": "^2.2.2",
```

Comment:
The library looks better maintained but this is a typescript project and i would suggest this library https://www.npmjs.com/package/browser-image-compression

Reply:
changed the library. good catch.